typescript/Linked Lists/lru_cache.ts

```diff
@@ -0,0 +1,73 @@
+import { DoublyLinkedListNode } from './ds';
```

Comment:
In this one, coding the data structure is part of the problem, so the class shouldn't be commented out

Reply:
oh shoot, you're right. it's not shared elsewhere in the chapter. thanks, buddy! fixed!